File: snippets/typescript/delete_payload.ts
import { QdrantClient } from "@qdrant/js-client-rest";

const client = new QdrantClient({ host: "localhost", port: 6333 });

client.deletePayload("global_patient_data", {
  keys: ["is_smoker", "notes"],
  points: [0, 3, 100],
});

File: snippets/go/delete_payload.go
package client

import (
	"context"

	"github.com/qdrant/go-client/qdrant"
)

func deletePayload() {
	client, err := qdrant.NewClient(&qdrant.Config{
		Host: "localhost",
		Port: 6334,
	})
	if err != nil {
		panic(err)
	}

	_, err = client.DeletePayload(context.Background(), &qdrant.DeletePayloadPoints{
		CollectionName: "{collection_name}",
		PointsSelector: qdrant.NewPointsSelector(qdrant.NewIDNum(0), qdrant.NewIDNum(3), qdrant.NewIDNum(100)),
		Keys:           []string{"color", "price"},
	})
	if err != nil {
		panic(err)
	}
}
